.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,25 @@
-<!-- Please take a look at our [Contributing](../blob/master/CONTRIBUTING.md)
-documentation before submitting a Pull Request!
-Thank you for contributing to Crossplane! -->
+<!--
+Please take a look at our [Contributing](../blob/master/CONTRIBUTING.md)
+documentation before submitting a pull request!
 
-**Description of your changes:**
+Thank you for contributing to Crossplane!
+-->
 
-**Which issue is resolved by this Pull Request:**
-Resolves #
+### Description of your changes
+<!--
+Briefly describe what this pull request does. Be sure to direct your reviewers'
+attention to anything that needs special consideration.
+-->
 
-**Checklist:**
-- [ ] Documentation has been updated, if necessary.
-- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
-- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
-- [ ] Code generation (`make generate`) has been run to update object specifications, if necessary.
-- [ ] CRD manifests generation (`make manifests`) has been run to update CRD manifests yaml file specifications, if necessary.
-- [ ] Update dependencies (`make vendor`) has been run to update `Gopkg.lock` file, if necessary
-- [ ] RBAC permissions in `clusterrole.yaml` have been updated to include new types, if necessary
-- [ ] All related commits have been squashed to improve readability.
-- [ ] Comments have been added or updated based on the standards set in [CONTRIBUTING.md](../blob/master/CONTRIBUTING.md#comments)
+Resolves # <!-- Add a Github issue here, like #123 -->
+
+### Checklist
+<!--
+Please run through this readiness checklist. 
+-->
+
+I have:
+- [ ] Run `make reviewable` to ensure this PR is ready for review.
+- [ ] Updated any relevant [documentation](docs/), [examples](cluster/examples/), or [release notes](PendingReleaseNotes.md).
```

Comment:
I'm not super sure whether these links will be relative to the root of the repo or relative to https://github.com/crossplaneio/crossplane/issues/new. 🤔

Reply:
I've seen links to `/LICENSE` and `../../pulls` work.

Reply:
Although it may be that you can only make it work in one case or the other -- preview mode while creating the PR, or when navigating the markdown preview of the .github/ directory.

Reply:
I updated them to go with `/foo` style links. I'll double check when if/when we merge this and update as necessary.

Reply:
these links work fine right now when looking at the "files changes" tab, right clicking each one and choosing "view file" to get the fully rendered markdown, and then clicking the links.